extensions/wikia/UserLogin/templates/UserLoginSpecial_Providers.php

```diff
@@ -1,4 +1,4 @@
-	<div class="sso-login">
+	<div class="sso-login fb-loaded hidden">
```

Comment:
Yeah, separating classes that are for function vs style is a good idea and we don't do much of that at wikia. Renaming would be fine. How about `fb-pending`? Then you can remove that class and the hidden class at the same time.